frontend/src/app/patients/ManagePatients.test.tsx

```diff
@@ -107,6 +107,19 @@ describe("ManagePatients", () => {
       ).toBeInTheDocument();
     });
 
+    it("when exiting archive modal, the action button is refocused", async () => {
+      render(<TestContainer />);
+      expect(await screen.findByText(patients[0].lastName, { exact: false }));
+
+      const menu = (await screen.findAllByText("More actions"))[0];
```

Comment:
Can you wrap these actions in the act method or switch them to fireEvent.click? It is to not get the act warnings . You can add an `.only` to the it method to only run the new added coverage `it.only(` so you can test it isolated!

Reply:
So I did a little bit of digging around this because it was annoying me and I think I found that the root cause is a package versioning mismatch. Expo [here](https://github.com/testing-library/user-event/issues/1104#issuecomment-1460005394). 

TL;DR is that our versions of `@testing-library/dom` need to match throughout the tree in order to not generate unnecessary act warnings.

Right now, running the yarn list on  `@testing-library/dom` gives us the following:

```
├─ @testing-library/dom@8.19.0
├─ @testing-library/react@14.0.0
│  └─ @testing-library/dom@9.3.0
└─ react-select-event@5.5.1
   └─ @testing-library/dom@9.3.0
```
Once I updated the root installation of `@testing-library/dom` to 9.3.0, the errors went away, even when using the `userEvent` action. For now, I just went ahead and replaced the one troublesome assertion with `fireEvent` so as to not add new errors, but hopefully the other errors will go away once we upgrade.

Reply:
ooo this is cool so if we upgrade @testing-library/dom to 9.3.0 the errors will get fixed in the other files as well?. I wonder why we haven't receive this upgrade from dependabot 🤔

Reply:
Yeah, when I updated locally the other errors in the file also went away. I'm guessing that it's in the queue per my question [here](https://skylight-hq.slack.com/archives/C01LTSNKEPP/p1686063839120479).

Reply:
<img width="999" alt="Screenshot 2023-06-06 at 10 42 17 AM" src="https://github.com/CDCgov/prime-simplereport/assets/29645040/cb4e68d1-e46b-4da4-a73c-f7e83751dabb">

looks like 9.3.0 was caught in yesterday's scan, so i'm guessing that i'll show up in the next round of updates